docs/src/main/tut/applicative.md

```diff
@@ -43,6 +43,6 @@ into the other context:
 Applicative functors are a generalization of Monads thus allowing to express 
 effectful computations into a pure functional way.
 
-Applicative functors are generally preferred to monads when the structure 
+Applicative functors are generally preferred to Monads when the structure 
```

Comment:
There's a tension here between talking about monads as concepts (lowercase) and the `Monad` type (uppercase). I think this should be lowercase.